cdp_scrapers/instances/houston.py

```diff
@@ -0,0 +1,283 @@
+import logging
+from datetime import datetime
+from typing import Any, List, Optional, Union
+
+import requests
+from bs4 import BeautifulSoup, NavigableString, Tag
+from cdp_backend.pipeline import ingestion_models
+from cdp_backend.pipeline.ingestion_models import EventIngestionModel
+
+from cdp_scrapers.scraper_utils import IngestionModelScraper
+
+log = logging.getLogger(__name__)
+
+
+class HoustonScraper(IngestionModelScraper):
+    def __init__(self):
+        super().__init__(timezone="America/Chicago")
+
+    def remove_extra_type(self, element: Union[Tag, NavigableString, None]) -> Tag:
+        """
+        Remove types that are not useful
+
+        Parameter:
+        ----------------
+        event: Union[Tag, NavigableString, None]
+            All elements in the page that we want to scrape
+
+        Returns:
+        ----------------
+        Tag
+            Same elements as received, assuming the elements are not null
+        """
+        if isinstance(element, NavigableString) or element is None:
+            raise ValueError(f"Wrong Type {type(element)}")
+        return element
+
+    def get_body_name(self, event: Union[Tag, NavigableString, None]) -> str:
+        """
+        Get the body name for an event
+
+        Parameter:
+        ----------------
+        event: Union[Tag, NavigableString, None]
+            All elements in the page that we want to scrape
+
+        Returns:
+        ----------------
+        str
+            The body name
+        """
+        log.info("start get body name")
+        event = self.remove_extra_type(event)
+        bodyTable = event.find_all("table")[1].find("table")
+        if "CITY COUNCIL" in bodyTable.text:
+            return "City Council"
+        else:
+            return bodyTable.find_all("span")[3].text.title()
+
+    def get_event_Minutes_Item(
```

Comment:
Prefer consistent casing; the letter `m` should be in lower case.

Reply:
Done